webpack.config.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path');
+
+module.exports = {
+  entry: './lib/index.ts',
+  resolve: {
+    extensions: ['.ts', '.js']
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(ts|js)?$/,
+        use: [
+          'babel-loader', {
+            loader: 'ts-loader',
```

Comment:
If you are using `babel-loader` _and_ `ts-loader` you don't need the babel typescript plugin (or vice-versa)